### PR TITLE
Implemented three new endpoints

### PR DESCRIPTION
**Pull request by marcusahl**
- Find if a word is in the training set
- Delete a word from the training set
- Retrieve the full set of training words

Updated readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,6 @@ Backend for Språkpolisen
 
 ## Usage
 Spin up the webserver by using `sbt run`, then access it in the browser at http://localhost:9000
+
+## Api endpoints
+See the conf/routes file
```
Comment (dirrelito) [inline README.md]: what file is that? would be super to reference the files by path :)

File: sprakpolisen/app/model/dao/wordEntryDAO.scala [hunk 2]
```diff
@@ -3,23 +3,54 @@ package model.dao
...
 
-//case class wordEntryDAO @Inject()(db: Database){
 object wordEntryDAO {  
+  
   def create(newWord: wordEntry) = {
     DB.withConnection() { implicit connection =>
       SQL(
           """
-          | INSERT IGNORE INTO `words` (`userId`, `word`)
+          | INSERT IGNORE INTO `words` (`languageId`, `word`)
```
Comment (dirrelito) [inline sprakpolisen/app/model/dao/wordEntryDAO.scala]: nice one! good to be future proof! :)

File: sprakpolisen/conf/routes [hunk 3]
```diff
@@ -6,11 +6,24 @@
 # API Endpoints
 # Retrieve the list of loaded training words
 #GET		/api/words/					actions.TODO.retrieve
-# Add a new word to the list of training words
-POST	/api/words/:word			actions.AddWord.add(word: String)
+
+# Word endpoints
+# Add a new word to the set of training words
```
Comment (dirrelito) [inline sprakpolisen/conf/routes]: fint du kommenterat häri!

**Review by el-hult — CHANGES_REQUESTED** [review]
Looks nice to my untrained eye. Overall very nice.
Clean format in the routes page with good comments.
Also good that you added language-support. Could be nifty in future!

One change request though... Can we exclude the 
Can we .gitignore the binary cache-file?

**Comment by marcusahl** [comment]
Added stuff to .gitignore

**Comment by el-hult** [comment]
Nice! Lets do it!

**Pull request merged**